runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -645,12 +645,16 @@ static bool owningMethodDoesNotContainBoundChecks(OMR::ValuePropagation *vp, TR:
    return false;
    }
 
-static TR::Node *getStoreValueBaseNode(TR::Node *storeValueNode, TR::SymbolReference *loadFlattenableElementSymRef)
+static TR::Node *getStoreValueBaseNode(TR::Node *storeValueNode,
+                                       TR::SymbolReferenceTable *symRefTab,
+                                       TR::SymbolReferenceTable::CommonNonhelperSymbol loadFlattenableArrayElementNonHelper,
+                                       TR::SymbolReference *loadFlattenableElementHelperSymRef)
```

Comment:
Given that `symRefTab` is an argument, maybe the arguments for `loadFlattenableArrayElementNonHelper` and `loadFlattenableElementHelperSymRef` aren't really needed anymore.  See comment at line 675 below.

Reply:
Addressed in 446e692